### PR TITLE
fixed sample package.json dependencies for challenge expand-your-project-with-external-packages-from-npm

### PR DESCRIPTION
**Pull request by JoelLau**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

**Comment by JoelLau** [comment]
## More Information

_Pull Request also fixes the examples found in other languages_

| Current | Fixed |
| ------- | ----- |
|  image | image |

**Comment by RandellDawson** [comment]
@JoelLau Why did you close your PR?

File: curriculum/challenges/english/05-apis-and-microservices/managing-packages-with-npm/expand-your-project-with-external-packages-from-npm.english.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ forumTopicId: 301527
 ---
```
Comment (thecodingaviator) [inline curriculum/challenges/english/05-apis-and-microservices/managing-packages-with-npm/expand-your-project-with-external-packages-from-npm.english.md]: &nbsp;

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Please take the changes made to this file off the PR i.e. revert it to its original state. Afaik, the formatting on the real challenge will remain unaffected by these changes and so they're moot.

**Review by thecodingaviator — COMMENTED** [review]
Thank you for opening this pull request.

This is a standard message notifying you that we’ve reviewed your pull request and have decided not to merge it. We would welcome future pull requests from you.

Thank you and happy coding. ✈️

**Pull request closed without merging**